at_commons/lib/src/keystore/at_key.dart

```diff
@@ -332,6 +334,7 @@ class Metadata {
       metaData.sharedKeyStatus = json[SHARED_KEY_STATUS];
       metaData.sharedKeyEnc = json[SHARED_KEY_ENCRYPTED];
       metaData.pubKeyCS = json[SHARED_WITH_PUBLIC_KEY_CHECK_SUM];
+      metaData.encoding = json['encoding'];
```

Comment:
change 'encoding' to a constant and declare it in the same place that SHARED_KEY_ENCRYPTED etc are declared

Reply:
Added constant in for 'encoding'